src/app/game-state/character.ts

```diff
@@ -498,13 +496,14 @@ export class Character {
     this.statLifespan = properties.statLifespan || 0;
     this.spiritualityLifespan = properties.spiritualityLifespan || 0;
     this.magicLifespan = properties.magicLifespan || 0;
-    this.attributeScalingLimit = properties.attributeScalingLimit;
-    this.attributeSoftCap = properties.attributeSoftCap;
-    this.aptitudeGainDivider = properties.aptitudeGainDivider;
     this.condenseSoulCoreCost = properties.condenseSoulCoreCost;
+    // This is derived to avoid save issues. Calculate rank and subtract from power to reduce the exponential aptitude divider.
+    this.aptitudeGainDivider = 5 * Math.pow(1.5, 9 - Math.log10(this.condenseSoulCoreCost / this.condenseSoulCoreOriginalCost)); 
     this.reinforceMeridiansCost = properties.reinforceMeridiansCost;
-    this.bloodlineCost = properties.bloodlineCost;
+    this.attributeScalingLimit = properties.attributeScalingLimit;
+    this.attributeSoftCap = properties.attributeSoftCap;
     this.bloodlineRank = properties.bloodlineRank;
+    this.bloodlineCost = 1000 * Math.pow(100, this.bloodlineRank); // This is derived to avoid save issues.
```

Comment:
Moved a lot around and derived values during the setProperties to avoid save issues. I recommend saving ranks for all three Ascensions rather than costs, but that might be hard to backcompat and it's not worth it at this point.

Reply:
Probably what I should have done in the first place. Yeah, not sure if it's useful now.

Reply:
I mean technically you could use the old costs to derive the ranks, save the ranks, and then derive new costs from the ranks, but again, the costs and uses seem to be pretty set in.